.github/workflows/dev-testing.yml

```diff
@@ -0,0 +1,110 @@
+name: Minimal and Full Tests
+on:
+  schedule:
+    - cron: "0 16 * * *"  # Daily at noon EST
+  workflow_call:
+    secrets:
+      DANDI_API_KEY:
+        required: true
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      S3_GIN_BUCKET:
+        required: true
+
+env:
+  DANDI_API_KEY: ${{ secrets.DANDI_API_KEY }}
+
+jobs:
+  run:
+    name: Minimal and full tests on ${{ matrix.os }} with Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: s-weigand/setup-conda@v1
+      - uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow --tags
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Global Setup
+        run: |
+          python -m pip install -U pip  # Official recommended way
+          pip install pytest-xdist
+          git config --global user.email "CI@example.com"
+          git config --global user.name "CI Almighty"
+          pip install wheel # Needed for scan image
```

Comment:
Should we move this line to the corresponding scan image requirements btw?

Reply:
A good question, but unfortunately no - I tried that back when this was added but it needs `wheel` installed _prior_ to installing dependencies from requirements/setup including the `ScanImageReader` package. Very silly, I know.

Reply:
What a shame indeed.